index.html

```diff
@@ -14,20 +14,172 @@
   <header class="header">
     <nav class="navbar">
       <a class="logo navbar-item position"><img src="images/image_geometry_1.svg" alt="A geometry image"/></a>
-      <i class="fas fa-bars navbar-item icon position"></i>
+      <i class="fas fa-bars navbar-item icon position primary-color"></i>
     </nav>
   </header>
   <section class="summary">
-    <div class="name position left16 align-left no-margin">
+    <div class="name position left16 align-left no-margin primary-color">
       <h1 class="no-margin">Ricky
       Mormor</h1>
     </div>
-    <div class="intro position left16 align-left width288">
+    <div class="intro position left16 align-left width288 poppins">
       <p>Hello! I am a software developer! I can help you build a product, feature or website. Taka a look of my works.
         If you like what you see and have a project you need coded, don’t hesitate and contact me.</p>
       </div>
 
-      <button class="cta position typography width288">Start collaboration</button>
+      <button class="cta position typography width288 primary-color">Start collaboration</button>
+  </section>
+
+  <section class="projects-section no-margin">
+    <div>
+      <img src="images/image-geometry_projects.png" class="head-image position" alt="A nice geometry image">
+    </div>
```

Comment:
- [ ] The above code for the background image is actually a decoration. Decorations are meant to be background images and not `img` tags. Please move this image url to the CSS stylesheet and asign it as a background image instead.